src/main/java/org/aksw/simba/squirrel/worker/impl/WorkerImpl.java

```diff
@@ -271,6 +273,12 @@ public void performCrawling(CrawleableUri uri, List<CrawleableUri> newUris) {
         LOGGER.debug("Fetched {} triples", count);
         setSpecificRecrawlTime(uri);
 
+        if (DeduplicatorComponent.deduplicationActive) {
+            //TODO: get triples somehow!
+            List<Triple> triples = new ArrayList<>();
+            TripleHashPostProcessor tripleHashPostProcessor = new TripleHashPostProcessor(this, triples, uri);
+            tripleHashPostProcessor.postprocess();
+        }
```

Comment:
This might slow down the worker a lot. I would prefer to have the hash computation somewhere else :thinking: